### PR TITLE
Mobile version skeleton of the Portfolio page

### PR DESCRIPTION
**Pull request by hamid-murambiwa**
This is a microverse project where I was required to create a single web page.
* Add index.html file
* Add *.css file
* Add README file
* Add linters for HTML and CSS
* Add .gitignore file
* Add .github folder

### CODE REVIEW

File: portfolio.css [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+.section-title {
+  font-size: 6.8vw;
+  text-align: center;
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  width: 100%;
+  color: #172b4d;
+  line-height: 12vw;
+}
```
Comment (dasileker) [inline portfolio.css]: - [x] The font size is bigger

File: portfolio.css [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+  width: 100%;
+}
+
+#s-1 p {
+  font-size: 5.5vw;
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  text-align: center;
+  color: #172b4d;
+}
```
Comment (dasileker) [inline portfolio.css]: - [x]  Add margin-top to make some space between the `span `and the  `p`

File: index.html [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+                <li>
+                    <a href="">
+                        <img src="images/medium.svg" alt="" class="social-icons">
+                    </a>
+                </li>
+
+                <li>
+                    <a href="">
+                        <img src="images/github.svg" alt="" class="social-icons">
```
Comment (dasileker) [inline index.html]: - [x] This is not a `nav` tag it should be `div class= "social-media"`, Please fix this

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @hamid-murambiwa**

### **To Highlight**
- The linters are passing.
- The commit msg are clear.
- The PR is descriptive.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: portfolio.css (hunk 1 above)
Comment (hamid-murambiwa) [reply portfolio.css]: How much bigger should the font size be.

**Review by carlos-ssh — APPROVED** [review]
Hi @hamid-murambiwa , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**